src/main/resources/application.properties

```diff
@@ -79,6 +84,9 @@ ccd.defaultPrintName=CCD Print
 ccd.defaultPrintDescription=Printing for CCD
 ccd.defaultPrintType=CCD Print Type
 
+idam.data-store.system-user.username=${DATA_STORE_IDAM_SYSTEM_USER_USERNAME:data.store.idam.system.user@gmail.com}
+idam.data-store.system-user.password=${DATA_STORE_IDAM_SYSTEM_USER_PASSWORD:Pa55word11}
```

Comment:
Think you already mentioned you were going to work on it, but reminder to update the chart values for these and the client secret.